arduino/temp-netsender/temp-netsender.ino

```diff
@@ -105,6 +110,17 @@ int tempReader(NetSender::Pin *pin) {
       pin->value = 10 * (ff + ZERO_CELSIUS);
       break;
     }
+  case 70: // TSL Lux
+    tsl.getEvent(&event);
+    if ((event.light == 0) |
+      (event.light > 4294966000.0) |
```

Comment:
More magic numbers; consider consts.

Reply:
The first thing I would do is to starting checking the return of ```tsl.getEvent(&event)```. This returns ```true``` on success and ```false``` otherwise. What you could be seeing is just garbage values.

Reply:
The other thing is that ```sensors_event_t.light``` is a ```float```, so if it is an unexpected overflow, it must be happening before the setting of the ```light``` field.